linux/keyman-config/keyman_config/install_window.py

```diff
@@ -296,7 +299,16 @@ def doc_policy(self, web_view, decision, decision_type):
     def on_install_clicked(self, button):
         logging.info("Installing keyboard")
         try:
-            install_kmp(self.kmpfile, self.online, language=self.language)
+            result = install_kmp(self.kmpfile, self.online, language=self.language)
+            if result:
+                dialog = Gtk.MessageDialog(
```

Comment:
```suggestion
                # If install_kmp returns a string, it is an instruction for the end user,
                # because for fcitx they will need to take extra steps to complete 
                # installation themselves.
                dialog = Gtk.MessageDialog(
```